README.md

```diff
@@ -123,6 +123,20 @@ slackSend(
 )
 ```
 
+#### Update Messages
+
+You can update the content of a previously sent message using the pipeline step.
+The step returns an object which you can use to retrieve the timestamp
+
+Example:
+
+```groovy
+def slackResponse = slackSend(channel: "updating-stuff", message: "Here is the primary message")
```

Comment:
why not support
```
slackResponse.updateMessage("yo dawg, I heard you like to update this message!")
```

Reply:
I think that sounds like a good idea, though i'm not familiar enough with the ins-and-outs of jenkins plugins to be certain of the best way to implement it in a DRY fashion. Would you be able to give me any direction on how to best utilize the existing `SlackSendStep` within the `SlackResponse` class in a jenkins-friendly fashion?

Reply:
I believe these response objects returned from pipeline steps are intended to be simple data objects without additional behaviour. Not 100% sure about that though!

Reply:
Nothing prevents you from having simple data object still with methods on them :)
You can override the toString to print of the object nicely.

Reply:
Would it be possible to land this and then consider adding that method? As-is, this implementation has functionality that allows full usage of the feature following current patterns, and that additional method would be some short-cutting that would require a bit of dev effort.

Reply:
Actually in #680 this was done with addReaction :)